### ISSUE TITLE
BE - how to update SVP site? (reactive)

### ISSUE THREAD

**Issue by lianilychee** [department-of-veterans-affairs/va.gov-team#26364]
# Description

**Question:** Hi Liani! We have some documentation updates to make, specifically to these ones here. Just confirming that all we need to do is update these docs in Confluence, and the changes automatically show on the platform website?  There’s probably a bunch of docs about Jenkins in github too. Do folks know to go look at either Confluence or the platform website now for the most up-to-date stuff, or are those not fully launched yet?

**Slack URL:** https://dsva.slack.com/archives/C01AS05BQ79/p1624374946070500

# Reference info

**Dates:** 6/21 asked, Date resolved to come.

**Days to resolution:** To come.

**Comment by lianilychee** [comment]
WIP, DRAFT: 

Hello Jeff!  Thanks for your patience - I wanted to make sure we got you a thorough and accurate answer.  And also for your q, because updating an existing page is exactly the gray area we were avoiding in our training session!

> Just confirming that all we need to do is update these docs in Confluence, and the changes automatically show on the platform website?

**Not quite - As Andrea said, changes to your 2nd link don't automatically render on the 1st link.  Part of the ["Create a page" training session (timestamp 11.14-14.02)](https://youtu.be/wYVbdIoEq8s?t=674) walks you through making sure there's parity between the 1st and 2nd links.  If you _don't_ have access to the Scroll Viewport admin dashboard shown in 11.26, lmk.**

> Do folks know to go look at either Confluence or the platform website now for the most up-to-date stuff, or are those not fully launched yet?

_Short answer:_ Not fully launched to VFS teams.  BUT we're also avoiding "fully launching," aka making a big bang announcement.

_Longer answer:_ 
- We're avoiding a big bang announcement because the Platform website is dependent on so many Platform teams that have variable plans.  (Such as: The eng teams are forecasting they'll conduct audits at the start of Q3, but only some experience teams will do so at the 2nd half of Q3, other experience teams have already conducted their audits, etc.)  
- Tentatively this quarter (pending finalizing roadmap), we are going to strategize how we can work the Platform website into orientation + how teams can socialize the Platform website along with any product launches (cuz product documentation).
  - We = Content, Service Design, Rachael Roueche
- What's happening now: I've shared the Platform website to onboarding VFS teams /team members, and I've seen indiv pages of the Platform website be socialized on a sitch-by-sitch basis to VFS team members, so it _has_ unofficially been making the rounds.
